teams_test.go

```diff
@@ -158,10 +158,13 @@ func TestAddTeam(t *testing.T) {
 	name := "TestTeam"
 	email := ""
 
-	err := client.AddTeam(name, email)
+	id, err := client.AddTeam(name, email)
 	if err != nil {
 		t.Error(err)
 	}
+	if id == 0 {
+		t.Error("AddTeam returned an invalid id.")
```

Comment:
Error strings shouldn't be punctuated:
```suggestion
		t.Error("AddTeam returned an invalid ID")
```